app/models/grda_warehouse/ch_enrollment.rb

```diff
@@ -301,10 +313,27 @@ def self.homeless_duration_sufficient(enrollment, date: enrollment.EntryDate)
     # Add steps for lines 15-16 and lines 22-23
     def self.length_of_stay_previous_sufficient(enrollment)
       steps = []
-      steps.push({ result: is_no?(enrollment.LOSUnderThreshold) || :continue, display_value: enrollment.LOSUnderThreshold })
+      value = enrollment.LOSUnderThreshold
+      result = if is_no?(value)
+        is_no?(value)
```

Comment:
Maybe
```suggestion
       :no
```
To be consistent?